website/docs/reference/database-permissions/about-database-permissions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Database permissions"
+id: about-database-permissions
+description: "Database permissions are access rights and privileges granted to users or roles within a database management system."
+sidebar_label: "About database permissions"
+---
+
+
+
+Database permissions are access rights and privileges granted to users or roles within a database or data platform. They help you specify what actions users or roles can perform on various database objects, like tables, views, schemas, or even the entire database.
+
+ARE THEY DIFF FROM MODEL GOVERNANCE BC CONTROLS THE DATABASE PERMISSIONS?
+
+## Why are they useful
+
+- Database permissions are essential for security and data access control.
+- They ensure that only authorized users can perform specific actions.
+- They help maintain data integrity, prevent unauthorized changes, and limit exposure to sensitive data.
+- Permissions also support compliance with data privacy regulations and auditing.
+
+## How to use them
```

Comment:
Is it worth adding here that permissions can be granted at various levels e.g. table, schema, warehouses, database etc. Also that each cloud vendor has a slightly different approach and naming to privelages so to check each vendors documentation for more specifics?

Reply:
great flag @rlh1994 , i've added some more context based on your sugg above!